plasmoid/contents/ui/ForecastDelegate.qml

```diff
@@ -23,11 +24,28 @@ Column {
     }
     
     PlasmaCore.IconItem {
+        visible: !useWxFonts
         source: icon
-        width: theme.smallMediumIconSize
+        width: useWxFonts ? 0 : theme.smallMediumIconSize
```

Comment:
Do we need to set width here? When `useWxFonts` is true, this item is not even visible.